api/clinics.go

```diff
@@ -83,7 +86,7 @@ func (h *Handler) CreateClinic(ec echo.Context) error {
 
 func (h *Handler) GetClinic(ec echo.Context, clinicId ClinicId) error {
 	ctx := ec.Request().Context()
-	clinic, err := h.Clinics.Get(ctx, string(clinicId))
+	clinic, err := h.ClinicsManager.GetWithPatientCounts(ctx, string(clinicId))
```

Comment:
This makes the clinic model inconsistent between the list and get endpoints (at the very least). If this is required by the client, then we should make it conditional on the presence of a new query parameter - e.g. `includePatientCounts=true`.

Reply:
Turns out this requirement was removed, I'll remove the code.